src/ui/editorwidgets/qgsexternalresourceconfigdlg.ui

```diff
@@ -632,7 +571,7 @@
  <resources/>
  <connections/>
  <buttongroups>
-  <buttongroup name="mStorageButtonGroup"/>
   <buttongroup name="mRelativeButtonGroup"/>
+  <buttongroup name="mStorageButtonGroup"/>
```

Comment:
Do you know what these refer to? I couldn't find the widgets in my Qt Designer (or I was too tired?)